action.yml

```diff
@@ -30,6 +30,10 @@ inputs:
     description: Show verbose output.
     required: false
     default: false
+  print_hash:
```

Comment:
I just realized that maybe it's better to make this plural. But it's too late now so I'll keep it as is.

Reply:
Maybe fix it in the next release?

Reply:
This would be a breaking change of the public API. We'd have to wait until v2 or even v3 depending on how we decide to do deprecation. There are other things, I'd change like using kebab-case for the inputs but not today.

Reply:
Yeah, and I just realized that.

Reply:
You may add a `print_hashes`, and treat the current name as an undocumented alias.